### DISCUSSION TITLE
Help with translation

### DISCUSSION THREAD

**Discussion by Clawhanx** [Muttley/foundryvtt-shadowdark#963] [General]
Howdy Muttley!

Is there a way to replace the STR, DEX, CON, etc texts with notepad+ ?
![image](https://github.com/user-attachments/assets/d5117d77-4f60-40aa-8c9c-164df626d778)
It is from the Character Generator/Creator, not the Character Sheet.

Also, these buttons?
![image](https://github.com/user-attachments/assets/03fda258-8ea1-4b7d-9393-4fa71295ce53)

I would be grateful if you could point out to me which files to edit.

**Comment by Muttley** [comment]
Hi,

All translation work needs to be done via the Crowdin project ( https://crowdin.com/project/foundryvtt-shadowdark ), and once changes are made there they will eventually end up in an automatic pull request and then pushed out with the next update.

If you need a new language added to the system, let me know and I can create it on Crowdin so it can be worked on.

Cheers

Muttley

**Reply by Clawhanx** [reply]
So it's not translatable, I believe? Because I'm using the pt_BR version and it didn't change the stats in the character generator, everything else inside the generator is translated though.

**Reply by Muttley** [reply]
If that's the case, then it's a bug.  If you raise an issue here then we can look at it for the next release: https://github.com/Muttley/foundryvtt-shadowdark/issues
